cpp/ql/test/library-tests/functions/constexpr/constexpr.ql

```diff
@@ -0,0 +1,6 @@
+import cpp
+
+from Function f
+select f, f.getFullSignature(),
```

Comment:
`getFullSignature` is deprecated since #823. It's deprecated only in the QLDoc comment for now, but we'll add a `deprecated` marker once the last users are removed from the internal repo.